cart.css

```diff
@@ -0,0 +1,82 @@
+@import url(/global-styles.css);
+
+.main{
+    background-color: var(--off-white);
+    height: 100vh;
+}
+
+.cart-body {
+    display: flex;
+    gap: 5rem;
+    justify-content: space-around;
+}
+.cart-title {
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 600;
+    padding: 1rem;
+}
+
+.card {
+    width: 20rem;
+    border: 1px solid var(--clr-black);
+}
+
+.card-title{
+    text-align: center;
+}
+.card-sub-title{
+    text-align: center;
+    
+}
+
+.card-quantity{
+    text-align: center;
+    font-size: 1.2rem;
+}
+.card-quantity i {
+    cursor: pointer;
+}
+
+.cart-price-content{
+    display: flex;
+    justify-content: space-between;
+}
+
+.product-card-button{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap:0.5rem;
+}
+.card-button-cart{
+    font-size: 1.2rem;
+    width: 100%;
+    border:none;
+    color: var(--clr-white);
+    border-radius: 1rem;
+    background-color: var(--cart-blue);
```

Comment:
color variable names can be more generalised, would be easy in case you change the color later.